### ISSUE TITLE
A11y Tab component Issues

### ISSUE THREAD

**Issue by mlmoravek** [oruga-ui/oruga#1127]
### Overview of the problem

**Oruga** version: [0.9.1]

The current Tab component have severall a11y issues which should be fixed.

### Expected behavior

No a11y issues when testing the Tabs docs page related to the Tabs component.

Sufficient axe tests are work with no problems.

### Actual behavior

**Elements may only use allowed ARIA attributes**
Ensures that ARIA attributes are not forbidden for an element's role.

Element Location:
_#tabpanel-1_

To solve this problem, you need to fix the following:
_aria-labelledby attribute cannot be used on div without valid role attribute._

[more information](https://dequeuniversity.com/rules/axe/4.10/aria-prohibited-attr?application=AxeChrome)

**Certain ARIA roles must contain specific child roles.**
Ensures that elements with an ARIA role that require certain child roles also contain them.

To solve this problem, you need to fix the following:
Element has children that are not allowed: [role=button]

[more information](https://dequeuniversity.com/rules/axe/4.10/aria-required-children?application=AxeChrome)

**Interactive controls must not be nested.**

Ensures that interactive controls are not nested, as they may not always be displayed by screen readers or may cause problems with focusing assistive technologies.

Element Location:
#tab-3

To solve this problem, you need to fix the following:
Element has focusable descendants.

[more information](https://dequeuniversity.com/rules/axe/4.10/nested-interactive?application=AxeChrome)
